fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -42,20 +42,31 @@ find_package(fastrtps REQUIRED CONFIG)
 string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastcdr_MAJOR_MINOR_VERSION "${fastcdr_VERSION}")
 string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastrtps_MAJOR_MINOR_VERSION "${fastrtps_VERSION}")
 
-find_library(FastRTPS_LIBRARY
+find_library(FastRTPS_LIBRARY_RELEASE
     NAMES fastrtps-${fastrtps_MAJOR_MINOR_VERSION} fastrtps)
 
-find_library(FastCDR_LIBRARY
+find_library(FastRTPS_LIBRARY_DEBUG
+    NAMES fastrtps${CMAKE_DEBUG_POSTFIX}-${fastrtps_MAJOR_MINOR_VERSION} fastrtps)
```

Comment:
If we can't resolve the problem with the broken nightly we might consider to revert the original change since the beta 2 freeze is approaching and I would rather have all jobs passing for that.